open_dread_rando/files/schema.json

```diff
@@ -36,20 +36,10 @@
                     "resources": {
                         "type": "array",
                         "items": {
-                            "type": "object",
-                            "properties": {
-                                "item_id": {
-                                    "$ref": "#/$defs/item_id"
-                                },
-                                "quantity": {
-                                    "type": "number"
-                                }
-                            },
-                            "required": [
-                                "item_id",
-                                "quantity"
-                            ],
-                            "additionalProperties": false
+                            "type": "array",
```

Comment:
Could you add a description to how exactly `resources` should be? Maybe use the `examples` to add a couple of cases so it's more obvious?